.github/workflows/ci.yml

```diff
@@ -168,13 +168,18 @@ jobs:
         run: . scripts/setenv -q && run-tests
       - name: Build artifacts
         shell: bash
-        run: . scripts/setenv -q && build-portable artifacts/OpenRCT2-MacOS-x64-cmake.tar.gz bin/install/usr
+        run: |
+          . scripts/setenv
+          mkdir -p artifacts
+          mv bin/OpenRCT2.app artifacts
+          echo -e "\033[0;36mCompressing OpenRCT2.app...\033[0m"
+          cd artifacts
+          zip -rqy openrct2-macos.zip OpenRCT2.app
```

Comment:
Why zip instead of tar?

Reply:
Simply matching the current implementation of the existing macOS CI (Xcode) job.